bbb_presentation_video/renderer/tldraw/utils.py

```diff
@@ -43,9 +43,9 @@ class SizeStyle(Enum):
 }
 
 FONT_SIZES: Dict[SizeStyle, float] = {
-    SizeStyle.SMALL: 28,
-    SizeStyle.MEDIUM: 48,
-    SizeStyle.LARGE: 96,
+    SizeStyle.SMALL: 25.5,
```

Comment:
How were these new font sizes determined?

The font sizes included in this file are [directly taken from tldraw upstream](https://github.com/tldraw/tldraw/blob/55a8831a6b036faae0dfd77d6733a8f585f5ae23/packages/tldraw/src/state/shapes/shared/shape-styles.ts#L100-L105). If we need to do any adjustments like scale by a fixed amount, that should be done in the rendering code. I would like to leave the constants matching so it's easier to keep these in sync.

The font sizes here are all around 91% to 92% smaller; we could probably multiply the sizes by a fixed factor in `renderer/tldraw/shape/text.py`.

Reply:
They were determined by visual ispection comparing the text in tldraw and the video rendered, but after seeing closely I found that what was wrong was the letter spacing of the font.
Pushed a fix reverting the font sizes and setting the letter spacing according to [tldraw](https://github.com/tldraw/tldraw/blob/main/packages/tldraw/src/constants.ts#L3)